GNUmakefile

```diff
@@ -35,17 +31,17 @@ install: fmtcheck
 
 .PHONY: test
 test: fmtcheck
-	go test $(TEST) -timeout=30s -parallel=4 -race -covermode=atomic -coverprofile=coverage.out
+	go test ./... -timeout=30s -parallel=4 -race
```

Comment:
just for reference I see this was included originally for in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1496, but we no longer include these converage reports as PR comments.

Reply:
yes, at the moment we don't include any coverage report